app/models/vacancy.rb

```diff
@@ -51,13 +51,14 @@ class Vacancy < ApplicationRecord
 
   scope :active, (-> { where(status: %i[published draft]) })
   scope :applicable, (-> { where("expires_at >= ?", Time.current) })
-  scope :expired, (-> { published.where("expires_at < ?", Time.current) })
   scope :awaiting_feedback, (-> { expired.where(listed_elsewhere: nil, hired_status: nil) })
+  scope :expired, (-> { published.where("expires_at < ?", Time.current) })
+  scope :expired_yesterday, (-> { where("expires_at BETWEEN ? AND ?", Time.zone.yesterday.midnight, Date.current.midnight) })
+  scope :in_organisation_ids, (->(ids) { joins(:organisation_vacancies).where(organisation_vacancies: { organisation_id: ids }).distinct })
   scope :listed, (-> { published.where("publish_on <= ?", Date.current) })
-  scope :pending, (-> { published.where("publish_on > ?", Date.current) })
   scope :live, (-> { listed.applicable })
+  scope :pending, (-> { published.where("publish_on > ?", Date.current) })
   scope :published_on_count, (->(date) { published.where(publish_on: date.all_day).count })
-  scope :in_organisation_ids, (->(ids) { joins(:organisation_vacancies).where(organisation_vacancies: { organisation_id: ids }).distinct })
```

Comment:
The above confusion of + and - is just
- Alphabetize
- Add `expired_yesterday` scope